extras/build/index.js

```diff
@@ -1,10 +1,19 @@
-require('./webfonts.js')
-require('./animate')
-
-require('./material-icons.js')
-require('./mdi-v5.js')
-require('./fontawesome-v5.js')
-require('./ionicons-v5.js')
-require('./eva-icons.js')
-require('./themify.js')
-require('./line-awesome.js')
+const parallel = require('os').cpus().length > 1
+const runJob = parallel ? require('child_process').fork : require
+const { join } = require('path')
+
+runJob(join(__dirname, './webfonts.js'))
+runJob(join(__dirname, './animate.js'))
+
+// replaced by material-icons2
+// runJob(join(__dirname, './material-icons.js'))
```

Comment:
Can we remove these kind of comments/dead code?
It's still possible to check out previous version using Git

Reply:
removed - thanks